src/plugins/exportAsJSONAction/ExportAsJSONAction.js

```diff
@@ -192,12 +219,39 @@ export default class ExportAsJSONAction {
                         this._saveAs(this._wrapTree());
                     }
```

Comment:
Can we move from `this.calls--;` to here into a separate function? `decrementCalls` or something? We use it in multiple places would be nice to have it in one spot.